cypress/support/step_definition/dashboard/dashboard.spec.js

```diff
@@ -0,0 +1,22 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import Dashboard from '../../page/common/dashboard'
+
+When('I arrive on the dashboard', () => {
+    const selectDashboard = new Dashboard()
```

Comment:
Same here. If we make it outside we done have to create the instance again and again and saves us some space.

Reply:
Fine for me. Maybe this is something we can work on, in the meeting on friday.

Reply:
Solved.